index.spec.js

```diff
@@ -18,6 +18,8 @@ describe('microargs', () => {
     expect(microargs(['--abc=123'])).toEqual({params: [], options: {abc: 123}})
     expect(microargs(['-a=http://www.google.com'])).toEqual({params: [], options: {a: 'http://www.google.com'}})
     expect(microargs(['--abc=http://www.google.com'])).toEqual({params: [], options: {abc: 'http://www.google.com'}})
+    expect(microargs(['-s=hello world'])).toEqual({params: [], options: {s: 'hello world'}})
+    expect(microargs(['--say=hello world'])).toEqual({params: [], options: {say: 'hello world'}})
```

Comment:
This is wrong, what if, as it's hard to say if `world` should part of `--say` or is it another param, for example:

```
script --abc=hello world test
```

should `abc` be equal `hello world` or `test` ?

Space for option value should be allowed only if enclosed in `"`

Reply:
It looks odd.. but this is because process.argv strips the quotes. I seem to remember that microargs takes in process.argv... unless i remember wrong (sorry if do!)

eg:
```
runjs hello --say="hello world" --a=b
console.log('argv', process.argv);
```
>  argv [ 'node.exe',  'run.js',  'hello',  '--say=hello world', '--a=b' ]

Reply:
@fromkeith yes that's true, `microargs` takes args from `process.argv` but in your example `process.argv` is equal this:

```
argv [ 'node.exe', 'run.js', 'hello', '--say=hello, 'world', '--a=b' ]
```

`argv` breaks args by spaces

Reply:
Here is an example I directly executed:

index.js
```
console.log('argv', process.argv);
```

bash (WSL):
```
node index.js --say="hello world" --a=b
```

output:
```
argv [ '/usr/local/bin/node',
  '/mnt/c/code/index.js',
  '--say=hello world',
  '--a=b' ]
```

So if microargs takes in ```process.argv```, then the quotes get removed automatically by node. 

(If it matters: Node v8.11.4. Tested on Windows and WSL)

Reply:
Ha, I tested it and it truly behaves in that way. I didn't know about that. Ok so that changes the situation

Reply:
I wonder only, what does this, shell or node

Reply:
True.. didn't think of that